HordaMaria/HW6/task2

```diff
@@ -0,0 +1,8 @@
+login = "First"
```

Comment:
good job
look at this code
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)